regolith/helpers/u_finishprumhelper.py

```diff
@@ -60,6 +64,16 @@ def db_updater(self):
             print("Please rerun the helper specifying the complete ID.")
             return
         found_projectum.update({'status':'finished'})
+        if rc.pub_sum:
+            found_projectum['deliverable'].update({'public_summary': rc.pub_sum})
+        elif found_projectum['deliverable'].get('public_summary') is not None:
```

Comment:
I think the logic should probably be:
```
if not found_prum['professional_summary']:
  if rc.pro_sum:
     found_prum['professional_summary'] = rc.pro_sum
  else:
     raise RuntimeError("ERROR: please rerun with a professional summary in field --pro_sum and public summary in --pub_sum")
```
and
```
if not found_prum['public_summary']:
  if rc.pub_sum:
     found_prum['professional_summary'] = rc.pub_sum
  else:
     raise RuntimeError("ERROR: please rerun with a professional summary in field --pro_sum and public summary in --pub_sum")
```

Reply:
Hi Simon, I am a bit hesitate to change the logic into this way, because once we update the `schema.py` and `a_projectumhelper.py`, when people create a new prum, the `professional_summary` field will automatically have some words there (because we initize it).

In this situation,  the first if statment`if not found_prum['public_summary']:` is useless.

Reply:
We should prevent people entering random stuff or not changing the original initialization words in the `professional_summary` field when people finish a prum.